setup.cfg

```diff
@@ -51,8 +51,6 @@ console_scripts =
 
 [options.extras_require]
 dev =
-    bumpver==2021.1114
```

Comment:
Unrelated change, I asked about this on Discourse.

Reply:
Hi @danielhollas, you add it back in the new commit correct?

Reply:
I did, since we didn't yet resolve this conversation on Discourse. (I still think it would be beneficial to remove, I'll reply on Discourse when I have time)

Reply:
Another option maybe don't pin down the version then if it already installed it will use it, otherwise install the latest version.

Reply:
I've unpinned them for now.